src/agri_ai/langchain_tools/field_info_tool.py

```
@@ -13,73 +13,104 @@
+
     async def _format_single_field(self, field: Dict[str, Any]) -> Dict[str, Any]:
         """単一圃場の情報をフォーマット"""
+        # 面積をha単位で表示
+        area_m2 = field.get("area", 0)
+        area_ha = field.get("area_ha", area_m2 / 10000 if area_m2 > 0 else 0)
+        
+        # ha単位を優先的に表示
+        if area_ha >= 0.01:  # 0.01ha以上ならha表示
+            area_display = f"{area_ha:.1f}ha"
+        else:
+            area_display = f"{area_m2}㎡"
```

Ruff (0.12.2)

Use ternary operator `area_display = f"{area_ha:.1f}ha" if area_ha >= 0.01 else f"{area_m2}㎡"` instead of `if`-`else`-block

(SIM108)